src/app/geschenke/page.tsx

```diff
@@ -15,8 +18,32 @@ export async function generateMetadata() {
   };
 }
 
+const styles = {
+  yearlyOnly: css({
+    display: "block",
+    ":has([value='ABO_GIVE_MONTHS']:checked) &": {
+      display: "none",
+    },
+  }),
+  monthlyOnly: css({
+    display: "none",
+    ":has([value='ABO_GIVE_MONTHS']:checked) &": {
+      display: "block",
+    },
+  }),
+};
+
 export default async function GiftsPage() {
   const t = await getTranslations("landing.gifts");
+  const tDescriptionItems = await getTranslations(
+    "landing.gifts.description.items"
+  );
+
+  const getText = (tKey: "dialog" | "general" | "briefings" | "goodie") =>
+    tDescriptionItems.rich(tKey, {
+      b: (chunks) => <b>{chunks}</b>,
+      p: (chunks) => <p className={css({ mt: "2" })}>{chunks}</p>,
+    });
```

Comment:
i would personally prefer this to be a part of the `DescriptionItem` logic, with key being passed as prop